src/env/node.js

```diff
@@ -63,14 +111,66 @@ export default {
 		process.env.NODE_ENV = "test";
 	},
 	done (result, options, event, root) {
-		let messages = root.toString({ format: options.format ?? "rich" });
-		let tree = getTree(messages).toString();
-		tree = format(tree);
-		logUpdate(tree);
+		makeCollapsible(root)
+		render(root, options);
 
 		if (root.stats.pending === 0) {
+
+			let hint = `
+<b>You are in interactive mode now.</b>
```

Comment:
I would remove this, no other interactive CLI app I've seen makes special mention of being in interactive mode, it's just the normal mode.

Reply:
Agreed